src/resolve-template.js

```diff
@@ -0,0 +1,90 @@
+var path = require("path");
+var fileSystem = require('fs');
+var readdir = require('recursive-readdir');
+var ContextElementDependency = require('webpack/lib/dependencies/ContextElementDependency');
+var assign = Object.assign || require('object.assign');
+
+var cheerio = require('cheerio');
+
+/**
+ * Generates key-value pairs or <require from="paths"> and their corresponding files.
+ */
```

Comment:
nice, code from this method 
_(iterating over all html files and checking that `from` attributes of `require` tags reference files, that exist in the project)_
could be taken as a base for implementing [Build-time validation (linter) for Aurelia application](https://github.com/aurelia/tools/issues/31), namely for
> typos with incorrect `<require from="./not/found">` element path